string id
string name
string value
string update_date